.github/workflows/build.yaml

```diff
@@ -0,0 +1,64 @@
+# Simple C#/.NET CI Build.
+# Test, that the repository contains a clean and working code.
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          fetch-tags: false
+
+      - name: Install .NET
+        uses: actions/setup-dotnet@v4
+        with:
+          dotnet-version: 8.0.x
+
+      - name: Test
+        run: |
+          cd ./TimeTracker
+          dotnet test --no-build --verbosity normal
+
+      - name: Build
+        run: |
+          cd ./TimeTracker
+          dotnet build
+
+      - name: Upload Artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: time-tracker-debug
+          path: bin
+
+  lintformat:
```

Comment:
why not make it part of the previous job?

Reply:
because i thought that the `test/build` could later become a mandatory job, the other one can have it's own reportr and a less mandatory build.